.env

```diff
@@ -103,6 +103,28 @@ MF_BOOTSTRAP_DB_PASS=mainflux
 MF_BOOTSTRAP_DB=bootstrap
 MF_BOOTSTRAP_DB_SSL_MODE=disable
 
+### Provision
+MF_PROVISION_CONFIG_FILE=/configs/config.toml
+MF_PROVISION_LOG_LEVEL=info
+MF_PROVISION_HTTP_PORT=8190
+MF_PROVISION_ENV_CLIENTS_TLS=false
+MF_PROVISION_CA_CERTS=
```

Comment:
Do we leave it like this on other services or we put explicitly empty string, i.e. `""`?

Reply:
yes

Reply:
Yes what?